windows-icon-handler/install_dll.bat

```diff
@@ -0,0 +1,2 @@
+regsvr32 /n /i:user .\x64\Debug\check-icon-handler\check-icon-handler.dll
+regsvr32 /n /i:user .\x64\Debug\refresh-icon-handler\refresh-icon-handler.dll 
```

Comment:
missing newline

Reply:
did you have `editorconfig` installed ?

Reply:
> did you have `editorconfig` installed ?

Yes

Reply:
That strange because vscode would have been correctly configured :thinking:

Reply:
I forgot to install it on windows that's why my first commits has these issues.

 >Yes, I just discovered that vscode does not add trailing newline by default :( I've configured it now. I'll add missing trainling > newline in a next commit

I just installed `editorconfig` and everything went back to normal 😄